extensions/interactions/DragAndDropSort/DragAndDropSort.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Python configuration for DragAndDropSort interaction."""
+
+from extensions.interactions import base
+
+
+class DragAndDropSort(base.BaseInteraction):
```

Comment:
@seanlip should we change its name to sorting interaction or something?
/cc @tjiang11 @kevinlee12